app/safe/internal/state/io/read.go

```diff
@@ -35,8 +35,8 @@ import (
 //     secret. If any error occurs during the process, a nil pointer and an error object are returned.
 //     The error provides context about the nature of the failure, such as issues with decryption or
 //     data deserialization.
-func ReadFromDisk(key string, rootKeyTriplet []string) (*entity.SecretStored, error) {
-	contents, err := crypto.DecryptDataFromDisk(key, rootKeyTriplet)
```

Comment:
now the root key triplet is a sruct, making it less error prone.

Also removed it from places that it is not directly needed.

Anytime a function needs the rkt it can just call `crypto.RootKeyTriplet()` or something like that.